manifests/k/Keboola/KeboolaCLI/2.1.1/Keboola.KeboolaCLI.locale.en-US.yaml

```diff
@@ -0,0 +1,23 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Keboola.KeboolaCLI
+PackageVersion: 2.1.1
+PackageLocale: en-US
+Publisher: Keboola Industries SE

```

Comment:
```suggestion
Publisher: Keboola Idustries SE
```
It will have to stay like this for now:
![image](https://user-images.githubusercontent.com/11600822/154465145-2d2c26a2-66d2-4508-90ae-91d5aad86e60.png)

You can contact the publisher to fix the typo.

Reply:
Right, I see we have the typo in our repo. Thanks.